.github/workflows/label_merge_conflicts.yml

```diff
@@ -0,0 +1,18 @@
+name: Manage labels
+
+on:
+  push:
+    branches:
+      - master
+  pull_request_target:
+    types: [synchronize]
+
+jobs:
+  label-conflicts:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Label merge conflicts
+        uses: eps1lon/actions-label-merge-conflict@v3
+        with:
+          dirtyLabel: "has conflicts"
+          repoToken: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
When triggered by `pull_request_target`, this token has write access. https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request_target
We should probably use a `permissions` node in this job to only grant the permissions that it actually needs (modifying pull request labels).

I don't know what permissions if any it will have when running on a push to master; another reason to specify exactly what permissions are being granted.

Reply:
I think the most granular I can get is `pull-requests: write`, as that's what enables editing labels. The only more fine-grained option I could find is with granular PATs, but that's not really an option here.

Reply:
https://docs.github.com/en/rest/issues/labels?apiVersion=2022-11-28#add-labels-to-an-issue

It needs `write` to both issues and pull requests, apparently.

Reply:
https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-pull-requests

https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-issues

Lists of what those permissions will allow.
I think explicitly:
```
issues: write
pull_requests: write
```
Is a lot better than whatever the token currently grants.